Watching for file changes with StatReloader
[15/May/2019 20:16:40] "GET / HTTP/1.1" 200 2053
[15/May/2019 20:16:40] "GET /static/css/base.css HTTP/1.1" 304 0
[15/May/2019 20:16:40] "GET /static/css/home.css HTTP/1.1" 304 0
[15/May/2019 20:19:34] "GET / HTTP/1.1" 200 2022
[15/May/2019 20:19:34] "GET /static/css/base.css HTTP/1.1" 200 1776
[15/May/2019 20:19:34] "GET /static/css/home.css HTTP/1.1" 304 0
[15/May/2019 20:20:45] "GET / HTTP/1.1" 200 2022
[15/May/2019 20:20:45] "GET /static/css/base.css HTTP/1.1" 200 1816
[15/May/2019 20:22:57] "GET / HTTP/1.1" 200 2050
[15/May/2019 20:22:57] "GET /static/css/base.css HTTP/1.1" 200 1930
[15/May/2019 20:22:57] "GET /static/css/home.css HTTP/1.1" 304 0
[15/May/2019 20:25:20] "GET / HTTP/1.1" 200 2050
[15/May/2019 20:25:20] "GET /static/css/base.css HTTP/1.1" 304 0
[15/May/2019 20:25:20] "GET /static/css/home.css HTTP/1.1" 304 0
[15/May/2019 20:26:24] "GET / HTTP/1.1" 200 2123
[15/May/2019 20:26:24] "GET /static/css/base.css HTTP/1.1" 200 1895
[15/May/2019 20:26:37] "GET / HTTP/1.1" 200 2123
[15/May/2019 20:26:37] "GET /static/css/base.css HTTP/1.1" 200 1915
[15/May/2019 20:29:15] "GET / HTTP/1.1" 200 2123
[15/May/2019 20:29:15] "GET /static/css/base.css HTTP/1.1" 200 1944
[15/May/2019 20:29:15] "GET /static/css/home.css HTTP/1.1" 304 0
[15/May/2019 20:29:25] "GET / HTTP/1.1" 200 2123
[15/May/2019 20:29:25] "GET /static/css/base.css HTTP/1.1" 200 1943
[15/May/2019 20:29:36] "GET / HTTP/1.1" 200 2123
[15/May/2019 20:29:36] "GET /static/css/base.css HTTP/1.1" 200 1942
[15/May/2019 20:30:10] "GET / HTTP/1.1" 200 2123
[15/May/2019 20:30:10] "GET /static/css/base.css HTTP/1.1" 200 1956
[15/May/2019 20:30:19] "GET / HTTP/1.1" 200 2123
[15/May/2019 20:30:19] "GET /static/css/base.css HTTP/1.1" 200 1956
[15/May/2019 20:32:33] "GET / HTTP/1.1" 200 2147
[15/May/2019 20:32:33] "GET /static/css/base.css HTTP/1.1" 304 0
[15/May/2019 20:32:33] "GET /static/css/home.css HTTP/1.1" 304 0
[15/May/2019 20:33:06] "GET / HTTP/1.1" 200 2147
[15/May/2019 20:33:06] "GET /static/css/base.css HTTP/1.1" 304 0
[15/May/2019 20:34:04] "GET / HTTP/1.1" 200 2147
[15/May/2019 20:34:04] "GET /static/css/base.css HTTP/1.1" 304 0
[15/May/2019 20:35:52] "GET / HTTP/1.1" 200 2223
[15/May/2019 20:35:52] "GET /static/css/base.css HTTP/1.1" 304 0
[15/May/2019 20:35:52] "GET /static/css/home.css HTTP/1.1" 304 0
[15/May/2019 20:37:06] "GET / HTTP/1.1" 200 2246
[15/May/2019 20:37:06] "GET /static/css/base.css HTTP/1.1" 304 0
[15/May/2019 20:37:52] "GET / HTTP/1.1" 200 2327
[15/May/2019 20:37:52] "GET /static/css/base.css HTTP/1.1" 304 0
[15/May/2019 20:39:13] "GET / HTTP/1.1" 200 2327
[15/May/2019 20:39:13] "GET /static/css/base.css HTTP/1.1" 304 0
[15/May/2019 20:39:13] "GET /static/css/home.css HTTP/1.1" 304 0
[15/May/2019 20:39:27] "GET / HTTP/1.1" 200 2333
[15/May/2019 20:39:35] "GET / HTTP/1.1" 200 2327
Exception in thread django-main-thread:
Traceback (most recent call last):
  File "/usr/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.6/threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "/home/scode/.local/lib/python3.6/site-packages/django/utils/autoreload.py", line 54, in wrapper
    fn(*args, **kwargs)
  File "/home/scode/.local/lib/python3.6/site-packages/django/core/management/commands/runserver.py", line 109, in inner_run
    autoreload.raise_last_exception()
  File "/home/scode/.local/lib/python3.6/site-packages/django/utils/autoreload.py", line 77, in raise_last_exception
    raise _exception[0](_exception[1]).with_traceback(_exception[2])
  File "/home/scode/.local/lib/python3.6/site-packages/django/utils/autoreload.py", line 54, in wrapper
    fn(*args, **kwargs)
  File "/home/scode/.local/lib/python3.6/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/scode/.local/lib/python3.6/site-packages/django/apps/registry.py", line 114, in populate
    app_config.import_models()
  File "/home/scode/.local/lib/python3.6/site-packages/django/apps/config.py", line 211, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/scode/.local/lib/python3.6/site-packages/django/contrib/auth/models.py", line 2, in <module>
    from django.contrib.auth.base_user import AbstractBaseUser, BaseUserManager
  File "/home/scode/.local/lib/python3.6/site-packages/django/contrib/auth/base_user.py", line 47, in <module>
    class AbstractBaseUser(models.Model):
  File "/home/scode/.local/lib/python3.6/site-packages/django/db/models/base.py", line 117, in __new__
    new_class.add_to_class('_meta', Options(meta, app_label))
  File "/home/scode/.local/lib/python3.6/site-packages/django/db/models/base.py", line 321, in add_to_class
    value.contribute_to_class(cls, name)
  File "/home/scode/.local/lib/python3.6/site-packages/django/db/models/options.py", line 204, in contribute_to_class
    self.db_table = truncate_name(self.db_table, connection.ops.max_name_length())
  File "/home/scode/.local/lib/python3.6/site-packages/django/db/__init__.py", line 28, in __getattr__
    return getattr(connections[DEFAULT_DB_ALIAS], item)
  File "/home/scode/.local/lib/python3.6/site-packages/django/db/utils.py", line 201, in __getitem__
    backend = load_backend(db['ENGINE'])
  File "/home/scode/.local/lib/python3.6/site-packages/django/db/utils.py", line 110, in load_backend
    return import_module('%s.base' % backend_name)
  File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/home/scode/.local/lib/python3.6/site-packages/django/db/backends/mysql/base.py", line 20, in <module>
    ) from err
django.core.exceptions.ImproperlyConfigured: Error loading MySQLdb module.
Did you install mysqlclient?

/home/scode/Capstone_Design/src/Django/scode/scode/views.py changed, reloading.
Watching for file changes with StatReloader
